construct.js

```diff
@@ -11,11 +12,18 @@ module.exports = (answers, templates, overwrite) => templates.forEach((template)
 
         ensureDirectories(destination);
 
-        if (!fs.existsSync(destination) || overwrite) {
+        if (!fs.existsSync(destination) || overwrite || template.overwrite) {
             const output = replace(template.template, answers, true);
 
             fs.writeFileSync(destination, output, { encoding: 'utf8' });
         }
+        else if(template.update) {
+            const templateObj = JSON.parse(replace(template.template, answers, true));
```

Comment:
I'd like to move this to it's own function--it's assuming that the only updateable files are JSON format--the new `update` function should either all the caller to specify their own update function (specified by `template.update`) or let the caller use pre-packaged messages, which can be specified by a string (aka, `'json`').